docs/api_endpoints.rst

```diff
@@ -61,6 +61,11 @@ Registration
 
     - key
 
+    .. note:: If you set account email verification as mandatory, you have to add the VerifyEmailView with the used `name`.
+        You need to import the view: ``from rest_auth.registration.views import VerifyEmailView``. Then add the url with the corresponding name:
```

Comment:
I think this should be `from dj_rest_auth.registration.views import VerifyEmailView`, not `from rest_auth`